src/facade/dragonfly_connection.cc

```diff
@@ -975,7 +975,7 @@ void Connection::DispatchSingle(bool has_more, absl::FunctionRef<void()> invoke_
         return;
 
       // Prefer sending synchronous request if possible (can_dispatch_sync=false),
-      // to reduce the memory pressure.
+      // to reduce the memory pressure. `has_more` is just an o
       has_more = false;
```

Comment:
you probably forgot something here 🙂

Reply:
i am rewriting this code again. btw, I just thought about a possible cool optimization to reduce hop pressure in dragonfly

Reply:
if thread_A schedules a hop on thread_B, and another connection schedules a hop on thread B, we use the same interpassing mechanism with lock-free queues. but we could chain our callbacks and reduce the message passing throughput.

Reply:
eh, it's probably equivalent to queues, nm